plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableDataStore.php

```diff
@@ -19,10 +21,19 @@ class OrdersTableDataStore extends \Abstract_WC_Order_Data_Store_CPT implements
 	 */
 	public function get_orders_table_name() {
 		global $wpdb;
-
 		return $wpdb->prefix . 'wc_orders';
 	}
 
+	public function get_addresses_table_name() {
+		global $wpdb;
+		return $wpdb->prefix . 'wc_order_addresses';
+	}
+
+	public function get_operational_data_table_name() {
+		global $wpdb;
+		return $wpdb->prefix . 'wc_order_operational_data';
+	}
+
 	// TODO: Add methods for other table names as appropriate.
 
 	//phpcs:disable Squiz.Commenting.FunctionComment.Missing
```

Comment:
Is it ok to do this? I thought it was a big sin in our codebase 🙂

Reply:
It's not ideal, but I am trying to figure out a good way for custom data stores to declare tables and schema. Bit of a WIP, so kept it as it is for now.